### ISSUE TITLE
docs: some imports are omitted

### ISSUE THREAD

**Issue by kuraga** [tkem/cachetools#295]
Hello! Thanks for your work!

Some imports are not displayed in [the docs](https://cachetools.readthedocs.io/en/v5.3.2/):
```python
import operator
from cachetools import cached, cachedmethod, LRUCache, TLRUCache, TTLCache
```
But they are used there.

P.S. Get rid of the one usage of the `operator`? It would be simpler.

**Comment by tkem** [comment]
@kuraga Thanks for the input! Examples generally assume `from cachetools import *`, I guess that's more or less in line with Python's standard docs, e.g. https://docs.python.org/3/library/functools.html
Regarding `operator`, that maybe should be stated explicitly. What's your issue with `operator`, i.e. how could this be made "simples"?

**Comment by kuraga** [comment]
@tkem ,
```python
@cachedmethod(lambda self: self.attrgetter('cache'))
```
is more understoodable (in that context) IMHO than
```python
@cachedmethod(operator.attrgetter('cache'))
```
and doesn't imply (another) implicit `import operator`.

**Comment by kuraga** [comment]
@tkem ,
```python
@cachedmethod(lambda self: self.cache)
```
is more understoodable (in that context) IMHO than
```python
@cachedmethod(operator.attrgetter('cache'))
```
and doesn't imply (another) implicit `import operator`.

**Comment by tkem** [comment]
@kuraga You may have a point here ;-) I'll think about it, thanks!

**Comment by tkem** [comment]
Would you care to make a pull request for that?

**Comment by kuraga** [comment]
'll make tomorrow both of them.

**Comment by kuraga** [comment]
> Would you care to make a pull request for that?
> 
> ```
> @cachedmethod(lambda self: self.cache)
> ```
> 
> Sounds pretty reasonable ;-)

Pushed in #297.

**Comment by tkem** [comment]
Merged #297 into master, thanks!
